programs/ssl/ssl_client2.c

```diff
@@ -515,6 +533,67 @@ int idle( mbedtls_net_context *fd,
     return( 0 );
 }
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+/*
+ * Load the key to a PSA key slot,
+ * then turn the PK context into a wrapper for that key slot.
+ *
+ * Currently only works for EC private keys.
+ */
+static int wrap_key_as_opaque( mbedtls_pk_context *pk, psa_key_slot_t *slot )
```

Comment:
@mpg @AndrzejKurek: As this function is also used in #2197, could we move it to the parent PR #2162 for both this and #2197 as a helper function in `pk_internal.h`?